### PR TITLE
change layout add avatar and split name from text

### PR DESCRIPTION
**Pull request by PamornT**
image

### CODE REVIEW

File: client/src/components/Message.js [hunk 1]
```diff
@@ -24,11 +31,19 @@ const Message = ({ message: { user, text }, name }) => {
...
+    <div class="grid grid-cols-6">
+      <div class="col-start-1 col-end-2 text-center">
+        <img class="inline object-cover w-8 h-8 rounded-full" src="https://p.kindpng.com/picc/s/78-785827_user-profile-avatar-login-account-male-user-icon.png" alt="Profile image"/><br/>
+        <div>
+          <p className="pr-2">{user}</p>
+        </div>
+      </div>
+      <div class="col-start-2 col-end-7">
+      <div className="flex flex-col items-start p-2 rounded-t-xl rounded-r-xl w-max max-w-xs bg-green-500 shadow-md text-white m-2 overflow-hidden lg:max-w-lg">
+          <p className="text-green-900 pr-2">{text}</p>
```
Comment (manuelalferez) [inline client/src/components/Message.js:43]: Please, change the color of the text: 
```
<p className="pl-2 text-green-900">{user}</p>
```
For
```
<p className="pl-2 text-green-100">{user}</p>
```
Left messages has a darker background:
Captura de pantalla 2021-10-06 a las 13 31 03

File: client/src/components/Message.js [hunk 2]
```diff
@@ -24,11 +31,19 @@ const Message = ({ message: { user, text }, name }) => {
...
+        </div>
       </div>
+
```
Comment (manuelalferez) [inline client/src/components/Message.js:46]: Could you format the document using Prettier? I totally forgot to mention it in the CONTRIBUTING. Sorry, my mistake. 
In that way every document meets the same guidelines.

**Comment by manuelalferez** [comment]
Thank you very much @PamornT, let me know if you need help with anything

File: client/src/components/Message.js (hunk 2 above)
Comment (PamornT) [reply client/src/components/Message.js]: Thank you for all your comment it good guideline for me

**Comment by PamornT** [comment]
> Thank you very much @PamornT, let me know if you need help with anything

Hi, I already fixed from your comment thank you very much :) it good repo for chat

**Pull request merged**